bootstrap-samples/src/main/java/de/agilecoders/wicket/samples/pages/ExtensionsPage.java

```diff
@@ -272,27 +274,31 @@ protected void onSubmit(AjaxRequestTarget target) {
                 sleep(3L);
             }
         };
-        laddaButton.setSize(Buttons.Size.Small);
+        spinnerButton.setSize(Buttons.Size.Small);
 
-        LaddaAjaxLink<String> laddaLink = new LaddaAjaxLink<>("laddaLink", null,
+        SpinnerAjaxLink<String> spinnerLink = new SpinnerAjaxLink<>("spinnerLink", null,
             Buttons.Type.Success, Model.of("Link, 2secs")) {
                 private static final long serialVersionUID = 1L;
 
             @Override
             public void onClick(AjaxRequestTarget target) {
-                sleep(3L);
+                sleep(2L);
             }
         };
-        laddaLink.setEffect(LaddaBehavior.Effect.SPINNER_BORDER).setSize(Buttons.Size.Medium);
+        spinnerLink.setEffect(SpinnerBehavior.Effect.SPINNER_BORDER).setSize(Buttons.Size.Medium);
 
-        form.add(new Code("linkCode", Model.of("laddaLink = new LaddaAjaxLink<String>(\"laddaLink\", Model.of(\"Link, 2secs\"), Buttons.Type.Success) {\n"
+        form.add(new Code("linkCode", Model.of("spinnerLink = new SpinnerAjaxLink<String>(\"spinnerLink\", null, Buttons.Type.Success, Model.of(\"Link, 2secs\")) {\n"
                                                + "    @Override public void onClick(AjaxRequestTarget target) {\n"
-                                               + "        Duration.seconds(2).sleep();\n"
+                                               + "        try {\n"
+                                               + "            Thread.sleep(Duration.ofSeconds(seconds).toMillis());\n"
+                                               + "        } catch (InterruptedException e) {\n"
+                                               + "            LOG.error(\"Sleep interrupted\", e);\n"
```

Comment:
Since this is an example for the end users it would be good to add `Thread.currentThread().interrupt();` here. Usually when an InterruptedException is caught it is a good practice to set the flag back to `true`.